build.gradle

```diff
@@ -24,6 +24,11 @@ repositories {
     jcenter()
```

Comment:
Unrelated to this PR, but JCenter no longer exists, you should probably remove this.